src/main/java/com/synopsys/integration/jira/common/cloud/model/request/IssueSearchRequestModel.java

```diff
@@ -89,11 +87,11 @@ public Boolean getFieldsByKeys() {
         return fieldsByKeys;
     }
 
-    private String initialValidationStrategy(final QueryValidationStrategy validationStrategy) {
+    private String initialValidationStrategy(QueryValidationStrategy validationStrategy) {
```

Comment:
Why are we removing final? was this a weird formatting change?

Reply:
@ekerwin says there is no longer a benefit to using it in method parameters and we should remove it wherever possible.